src/test/java/org/opentripplanner/model/plan/TestItineraryBuilder.java

```diff
@@ -145,9 +145,29 @@ public TestItineraryBuilder bus(
     int toStopIndex,
     Place to,
     LocalDate serviceDate
+  ) {
+    return bus(
+      Integer.toString(tripId),
+      startTime,
+      endTime,
+      fromStopIndex,
+      toStopIndex,
+      to,
+      serviceDate
+    );
+  }
+
+  public TestItineraryBuilder bus(
+    String tripId,
+    int startTime,
+    int endTime,
+    int fromStopIndex,
+    int toStopIndex,
+    Place to,
+    LocalDate serviceDate
```

Comment:
The `bus` and `transit` methods are duplicated - not DRY, just to change the type one argument (tripId). The design here is not very flexible, with static methods with a lot of arguments. Using a builder pattern is probably better, but that is a bigger refactoring. If you can convert you test to use numerical tripIds, that is probably the easiest solution.

Reply:
I've managed to cut down on the duplication in the commit 5213c136a038aa615bc36c21bbc84cb8f77208e9

Could you take a look if that is better? Unfortunately, this calculator needs to inspect the trip id.

Reply:
I think that is ok.